### ISSUE TITLE
convert tf mobilenet 2 to caffe,error

### ISSUE THREAD

**Issue by pageedward** [Microsoft/MMdnn#406]
Platform (like ubuntu 16.04/win10):

Python version: 2.7.14

Source framework with version (like Tensorflow 1.4.1 with GPU): tensorflow 1.8 GPU

Destination framework with version (like CNTK 2.3 with GPU): caffe 1

Pre-trained model path (webpath or webdisk path): http://download.tensorflow.org/models/mobilenet_v1_2018_02_22/mobilenet_v1_1.0_224.tgz

Running scripts:
mmconvert -sf tensorflow -iw mobilenet_v2_1.0_224_frozen.pb --inNodeName input --inputShape 224,224,3 --dstNodeName MobilenetV2/Predictions/Softmax -df caffe -om recognizer

problem as followed：
![image](https://user-images.githubusercontent.com/26244036/45337468-ccd73300-b5ba-11e8-8169-0420ba9dbc40.png)

**Comment by namizzz** [comment]
Hi @pageedward ,please try the newest version of MMdnn! Thanks!

**Comment by pageedward** [comment]
thanks ,it works now... 
is relu6 in caffe implemented ?

**Comment by namizzz** [comment]
Hi @pageedward ,no .But there are some third party caffe including relu6.
